client/src/pages/private/PSIView.js

```diff
@@ -213,35 +213,13 @@ export default () => {
               permittedRoles={['ministry_of_health', 'health_authority']}
             >
               <Box alignSelf='flex-end' py={1}>
-                {/** TODO: add in a dropdown once we have multiple download options
-                  <ClickAwayListener onClickAway={() => setDownloadMenuOpen(false)}> */}
                 <Button
-                  /** TODO: add in a dropdown once we have multiple download options
-                    onClick={() => setDownloadMenuOpen(!isDownloadMenuOpen)}
-                    endIcon={<ExpandMoreIcon />}
-                    text='Download CSV' 
-                  */
-                  text='Download participants attending PSI Report'
+                  startIcon={<GetAppIcon />}
+                  text='Download PSI list (.csv)'
```

Comment:
I didn't see this in any AC for HCAP-1039

Reply:
This is just to match the mockups we have

<img width="500" alt="Screen Shot 2022-09-08 at 2 29 20 PM" src="https://user-images.githubusercontent.com/64768811/189229483-36191abf-85e5-4348-8eb5-dc73028fd0d7.png">